config/passport.js

```diff
@@ -294,6 +294,74 @@ passport.use(
   ),
 );
 
+/**
+ * Sign in with Imgur.
+ */
+const imgurStrategyConfig = new OAuth2Strategy(
+  {
+    authorizationURL: 'https://api.imgur.com/oauth2/authorize',
+    tokenURL: 'https://api.imgur.com/oauth2/token',
+    clientID: process.env.IMGUR_CLIENT_ID,
+    clientSecret: process.env.IMGUR_CLIENT_SECRET,
+    callbackURL: `${process.env.BASE_URL}/auth/imgur/callback`,
+    state: generateState(),
+    passReqToCallback: true,
+  },
+  async (req, accessToken, refreshToken, params, profile, done) => {
+    try {
+      // Get user profile from Imgur API
+      const response = await fetch('https://api.imgur.com/3/account/me', {
+        headers: {
+          Authorization: `Bearer ${accessToken}`,
+        },
+      });
+
+      if (!response.ok) {
+        throw new Error('Failed to fetch Imgur user profile');
+      }
+
+      const userData = await response.json();
+      const imgurProfile = userData.data;
+
+      if (req.user) {
+        const existingUser = await User.findOne({
+          imgur: { $eq: imgurProfile.id },
+        });
+        if (existingUser && existingUser.id !== req.user.id) {
+          req.flash('errors', {
```

Comment:
Since we are not doing sign in by Imgur, you don't need to worry about email address, similar to how the traktv example is set.